<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite name="Test" tests="2" failures="1" timestamp="25 Jun 2014 16:26:34 GMT" time="14.611" errors="0">
  <testcase name="@AfterClass closeBrowser" time="0.0" classname="com.autotesting.framework.suits.invalidLoginName">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.autotesting.framework.suits.invalidLoginName.closeBrowser(invalidLoginName.java:21)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass closeBrowser -->
  <testcase name="invalidLoginNameToMail" time="6.985" classname="com.autotesting.framework.suits.invalidLoginName"/>
  <testcase name="validLogin" time="6.346" classname="com.autotesting.framework.suits.ValidLogin">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.autotesting.framework.suits.ValidLogin.validLogin(ValidLogin.java:15)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- validLogin -->
</testsuite> <!-- Test -->
